.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,104 @@
+
+name: "Continuous Integration"
+
+on:
+  pull_request:
+    branches:
+      - "*.x"
+      - "master"
+  push:
+    branches:
+      - "*.x"
+      - "master"
+
+env:
+  fail-fast: true
+
+jobs:
+  phpunit:
+    name: "PHPUnit"
+    runs-on: "ubuntu-20.04"
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.1"
+          - "7.2"
+          - "7.3"
+          - "7.4"
+          # - "8.0"
```

Comment:
@alcaeus can you help with this? It fails because `alcaeus/mongo-php-adapter` cannot be installed, what should be done here?

Reply:
That's odd, because 1.2.0 adds compatibility with PHO 8.0 🤔

Reply:
Let me enable it again so that we can see the failure.

Reply:
Ok I read to fast, looks like I just need to install the `mongo` extension

Reply:
You need to install the `mongodb` extension - the legacy driver (a.k.a. the `mongo` extension) was never ported for PHP 7, which is why the adapter package exists.

Reply:
I think this is what I did although I said otherwise :) I copied what you did in the ODM repo IIRC